src/conveyor/conveyor/conveyor_node.py

```diff
@@ -0,0 +1,96 @@
+# This ROS 2 node contains code for the conveyor subsystem of the robot.
+# Original Author: Anthony Brogni <brogn002@umn.edu> in Fall 2023
+# Maintainer: Anthony Brogni <brogn002@umn.edu>
+# Last Updated: September 2023
+
+# Import the ROS 2 Python module
+import rclpy
+from rclpy.node import Node
+
+# Import custom ROS 2 interfaces
+from rovr_interfaces.srv import MotorCommandSet
+from rovr_interfaces.srv import ConveyorSetPower, Stop
+
+# Define helper functions here
+def clamp(number: float, minimum: float, maximum: float) -> float:
+    """Clamps a number between the specified minimum and maximum."""
+    return max(minimum, min(number, maximum))
+
+
+class ConveyorNode(Node):
+    def __init__(self):
+        """Initialize the ROS 2 conveyor node."""
+        super().__init__("conveyor")
+        
+        # Define service clients here
+        self.cli_motor_set = self.create_client(MotorCommandSet, "motor/set")
+        
+        # Define services (methods callable from the outside) here
+        self.srv_toggle = self.create_service(ConveyorSetPower, 'conveyor/toggle', self.toggle_callback)
+        self.srv_stop = self.create_service(Stop, 'conveyor/stop', self.stop_callback)
+        self.srv_setPower = self.create_service(ConveyorSetPower, 'conveyor/setPower', self.set_power_callback)
+        
+        # Define motor CAN IDs here
+        self.DRUM_BELT_MOTOR = 7
+        self.CONVEYOR_BELT_MOTOR = 6
+        
+        # Current subsystem state
+        self.running = False
+
+    # Define subsystem methods here
+    def set_power(self, drum_belt_power: float, conveyor_belt_power: float) -> None:
+        """This method sets power to the conveyor belts."""
+        self.running = True
+        drum_belt_power = clamp(drum_belt_power, -1.0, 1.0)  # Clamp the power between -1.0 and 1.0
+        conveyor_power = clamp(conveyor_belt_power, -1.0, 1.0)  # Clamp the power between -1.0 and 1.0
+        self.cli_motor_set.call_async(MotorCommandSet.Request(type="duty_cycle", can_id=self.DRUM_BELT_MOTOR, value=-1*drum_belt_power))
```

Comment:
Ideally don't just multiply by -1, you can make an inverse constant, but ideally this is reversed in vesc config

Reply:
This can be a new ticket